src/pages/all-on-one-v2/all-on-one-v2.ts

```diff
@@ -113,35 +122,77 @@ export class AllOnOneV2Page {
     return this.selectedManoeuvre === key;
   }
 
-  controlledStopPress() {
-    if (this.isControlledStopDone) return;
-    const { serious, dangerous } = this.controlledStopEl;
-    if (this.controlledStopEl.faultCounter > 0) {
+  // Show me button actions
+
+  setUpShowMeButton(tellMeFault: string) {
+    const noFault = undefined;
+    if (tellMeFault === noFault) {
       return;
     }
-    if (serious || dangerous) {
-      if (!this.isControlledStopDone) {
-        this.isControlledStopDone = !this.isControlledStopDone;
-        this.summaryMetaDataService.toggleControlledStopComplete();
-      }
+
+    const drivingFault = 'fault';
+    if (tellMeFault === drivingFault) {
+      this.showMeEl.addDrivingFault();
       return;
     }
+
+    const seriousFault = 'serious';
+    if (tellMeFault === seriousFault) {
+      this.showMeEl.addSeriousFault();
+      return;
+    }
+
+    const dangerousFault = 'dangerous';
+    if (tellMeFault === dangerousFault) {
+      this.showMeEl.addDangerousFault();
+      return;
+    }
+  }
+
+  showMePress() {
+    if (this.isShowMeDone) return;
```

Comment:
Does this lint? It probably shouldn't.

Reply:
Used the `git hook` on `commit` and it was not complaining.